lib/scout_apm/instruments/samplers/memory.ex

```diff
@@ -0,0 +1,30 @@
+defmodule ScoutApm.Instruments.Samplers.Memory do
+  def metrics do
+    value = as_duration()
+    [
+      %ScoutApm.Internal.Metric{
+        type: "Memory",
+        name: "Physical",
+        call_count: 1,
+        total_time: value,
+        exclusive_time: value,
+        min_time: value,
+        max_time: value,
+      }
+    ]
+  end
+
+  # Hacky right now...we are really only handling timing metrics. This basically sends up the value in MB
+  # exactly as collected with no units conversion. Server-side, we expect memory to be in MB.
+  def as_duration do
+    ScoutApm.Internal.Duration.new(total_mb(), :seconds)
+  end
```

Comment:
`defp`, and I'd have this take a value, and just wrap it as seconds. (`defp as_duration(val), do: Duration.new(val, :seconds)` and just pass the total_mb() value in from line 3. Keeps this hack of a function from knowing where that data came from.